platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -450,21 +534,27 @@ class HangingProtocolService {
           seriesMatchDetails.score + studyMatchDetails.score;
 
         const imageDetails = {
-          StudyInstanceUID: study.StudyInstanceUID,
-          SeriesInstanceUID: aSeries.SeriesInstanceUID,
+          StudyInstanceUID,
+          SeriesInstanceUID,
+          displaySetInstanceUID,
           matchingScore: totalMatchScore,
           matchDetails: matchDetails,
           sortingInfo: {
             score: totalMatchScore,
             study: study.StudyInstanceUID,
-            series: parseInt(aSeries.SeriesNumber),
+            series: parseInt(displaySet.SeriesNumber),
           },
         };
 
+        // console.log("Adding display set", displaySet, imageDetails);
         matchingScores.push(imageDetails);
       });
     });
 
+    if (matchingScores.length === 0) {
+      console.log("No match found");
```

Comment:
This is so weird in the console as there is no context. Maybe `HangingProtocolService:: No Match found for the specified rules`?